src/main/resources/reference.conf

```diff
@@ -1,7 +1,13 @@
 kamon {
   jaeger {
+    # If spans should be sent to jaeger agent (UDP/Thrift compact) instead of the collector (HTTP(S)/Thrift binary).
+    use-agent = false
+
     host = "localhost"
     port = 14268
+    agent-port = 6831
```

Comment:
Copying from here: https://gitter.im/kamon-io/Kamon?at=5d38f3acb0650861254141e4

It would be nice to have something similar to what we have in Zipkin now: only one host/port setting and a separate setting for the protocol, which in this case could be http/https?/udp. WDYT?

Reply:
Folks, I'm going to apply this little change and merge/release.. out of the upgraded modules, this is the only one missing for 2.0!